include/El/core/Grid.hpp

```diff
@@ -132,6 +134,92 @@ inline void AssertSameGrids( const Grid& g1, const Grid& g2, Args&... args )
     AssertSameGrids( g2, args... );
 }
 
+inline bool GridCompare
```

Comment:
I created grid level by level in my hierarchical structure.
The grid created in the constructer does not work for Matrix Combination. However, the local grid based on the same Comm and Group works. That's the reason for me to added this routine to check if these two grids are exactly the same or not.
The current GridCompare seems not returning the correct answer for the following piece of code:
Grid g1(comm, group, height);
Grid g2(comm, group, height);
GridCompare(g1,g2); // this return false.